appeals/web/src/server/appeals/appeal-details/appeal-details.mapper.js

```diff
@@ -358,6 +371,16 @@ function mapStatusDependentNotifications(appealDetails, session, accordionCompon
 				)}">Issue decision</a></p>`
 			);
 			break;
+		case 'lpa_questionnaire_due':
+			addNotificationBannerToSession(
+				session,
+				'appealValidAndReadyToStart',
+				appealDetails.appealId,
+				`<p class="govuk-notification-banner__heading">Appeal valid</p><p><a class="govuk-notification-banner__link" href="${generateStartTimetableUrl(
+					appealDetails.appealId
+				)}">Start case</a></p>`
+			);
```

Comment:
Doesn't this stay in session meaning that the notification will stay even after the case is no longer ready to start?
There is a remove function on ln 402, you may need to do the same for ready to start?